CorsixTH/Src/sdl_core.cpp

```diff
@@ -348,10 +348,22 @@ static int l_get_ticks(lua_State *L)
     return 1;
 }
 
+static int l_start_text_input(lua_State *L)
+{
+    SDL_StartTextInput();
+}
```

Comment:
Given that it doesn't push anything, I guess this is missing "return 0;" to fix appveyor error